recipe/build.sh

```diff
@@ -10,5 +10,4 @@ fi
 
 
 rm -r "${PREFIX}/doc"
-rm -r "${PREFIX}/extdata"
```

Comment:
Do we want to be packaging the test data?

Reply:
yeah. this is actually useless. my bad.

Reply:
No worries. It's easy to remove.